modules/hal_silabs/wiseconnect/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set(SISDK_DIR ${ZEPHYR_HAL_SILABS_MODULE_DIR}/simplicity_sdk)
+set(WISECONNECT_DIR ${ZEPHYR_HAL_SILABS_MODULE_DIR}/wiseconnect)
+
+if(CONFIG_PSA_CRYPTO_DRIVER_SILABS_SI91X)
+
+zephyr_library_named(hal_silabs_si91x_crypto)
```

Comment:
In Zephyr, we tends to use `siwx91x` rather than `si9x` (even if we are not aligned with Wiseconnect :-/ ).

Reply:
Done